### PR TITLE
Upgrade gradle

### PR DESCRIPTION
**Pull request by armendh**

### CODE REVIEW

**Comment by stevenzeck** [comment]
Hi @armendh, this looks better now.  As I mentioned in an earlier comment, this most likely would be merged right before or right after a Readium release.  Just a few things:

1. In `pdfium-document`, `pdfium-navigator`, `pspdfkit-document`, and `pspdfkit-navigator`:  Can you update `coroutines-android` and `coroutines-core` to 1.6.3 and `fragment-ktx` to 1.5.0?  Or if you already did that and ran into an issue, disregard.
2. In [MediaNavigator](https://github.com/readium/kotlin-toolkit/blob/develop/readium/navigator-media2/src/main/java/org/readium/navigator/media2/MediaNavigator.kt#L429), can you change that `setContentType` to `C.AUDIO_CONTENT_TYPE_MUSIC`?

File: readium/opds/build.gradle.kts [hunk 1]
```diff
@@ -42,14 +42,16 @@ android {
...
 afterEvaluate {
-    publishing {
-        publications {
-            create<MavenPublication>("release") {
-                from(components.getByName("release"))
-                groupId = "com.github.readium"
-                artifactId = "readium-opds"
-                artifact(tasks.findByName("sourcesJar"))
-                artifact(tasks.findByName("javadocsJar"))
+    if(rootProject.name == "Readium"){
```
Comment (mickael-menu) [inline readium/opds/build.gradle.kts:45]: Why did you add this conditional here?

Comment (armendh) [reply readium/opds/build.gradle.kts:45]: because it causes an error "Cannot convert a null value to an object of type MavenArtifact" when importing readium locally to a project (https://github.com/readium/kotlin-toolkit#from-a-local-git-clone)

**Comment by stevenzeck** [comment]
@armendh Sorry, can you make that same change in [ExoMediaPlayer](https://github.com/armendh/kotlin-toolkit/blob/3f273b573f159ad573d1735f2a57002245ec9c34/readium/navigator/src/main/java/org/readium/r2/navigator/media/ExoMediaPlayer.kt#L95)?  `C.AUDIO_CONTENT_TYPE_MUSIC`

**Comment by mickael-menu** [comment]
@qnga Finalized this PR in https://github.com/readium/kotlin-toolkit/pull/282. Thanks @armendh for kickstarting it.
